src/main/java/org/asamk/signal/manager/Manager.java

```diff
@@ -810,10 +810,12 @@ public String getGroupName(byte[] groupId) {
     /**
      * Change the expiration timer for a contact
      */
-    public void setExpirationTimer(SignalServiceAddress address, int messageExpirationTimer) {
-        ContactInfo c = account.getContactStore().getContact(address);
+    public void setExpirationTimer(String number, int messageExpirationTimer) throws InvalidNumberException {
+        ContactInfo c = account.getContactStore().getContact(canonicalizeAndResolveSignalServiceAddress(number));
         c.messageExpirationTime = messageExpirationTimer;
+        System.err.println("Set message expiration " + c.number + " -> " + c.messageExpirationTime + " seconds");
```

Comment:
I'd like to get rid of all console output in the Manager class, so please remove the println here as well.
As long as a command completes successfully, I'd prefer to not output anything as long as it doesn't provide additional information.

Reply:
Sure this makes sense, removed it